### ISSUE TITLE
InteractiveBrowserCredential cannot open browser in Windows WSL

### ISSUE THREAD

**Issue by heavytail** [Azure/azure-sdk-for-python#8859]
When using InteractiveBrowserCredential in Windows WSL, the process gets stuck at the get_token() step without opening a browser tab.

How to reproduce:
```
>>> from azure.identity import InteractiveBrowserCredential
>>> ibc = InteractiveBrowserCredential()
>>> ibc.get_token()
```

Environment:
Windows 10 1903
WSL 1 with Ubuntu 18.04
Python 3.6.8

**Comment by chlowell** [comment]
Thanks for reporting this! It happens because the credential uses the standard library's `webbrowser` to open a browser, which fails in WSL. The credential, however, continues waiting for a login until it times out. In the next release of azure-identity, `InteractiveBrowserCredential` will instead raise `ClientAuthenticationError` when `webbrowser.open` fails (#8465).

**Comment by chlowell** [comment]
The fix is now available, in [azure-identity 1.1.0](https://pypi.org/project/azure-identity/1.1.0/). The credential will still fail in WSL (and anywhere else `webbrowser.open` fails) but it will now do so immediately with an exception rather than waiting to time out.
